ui/src/styles/var/_padding.scss

```
@@ -0,0 +1,21 @@
+$pad-xxsmall: px-to-rem(2);
+$pad-xsmall: px-to-rem(4);
+$pad-small: px-to-rem(8);
+$pad-smedium: px-to-rem(12);
+$pad-icon: px-to-rem(14);
+$pad-medium: px-to-rem(16);
+$pad-large: px-to-rem(24);
+$pad-xlarge: px-to-rem(32);
+$pad-xxlarge: px-to-rem(40);
+$pad-xxxlarge: px-to-rem(80);
+
+$pad-page: $pad-xlarge;
```

Stylelint (17.7.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)